doc/source/getting_started/develop_pymapdl.rst

```diff
@@ -0,0 +1,337 @@
+
+
+While the code coverage in either case is 100% for the function, the second case is
+more useful for debugging the function.
+
+
+Continuous Integration and Continuous Deployment (CI/CD) approach
+-----------------------------------------------------------------
+
+Unit tests and integration tests are part of Continuous Integration (CI). 
+The automation of testing, monitoring, and deployment of newly added
+code allows Continuous Deployment (CD) throughout the app lifecycle,
+providing a comprehensive CI/CD approach.
+
+.. figure:: ../images/cicd.jpg
+    :width: 300pt
+
+Creation of a unit test
+-----------------------
+
+In the PyMAPDL repository, `pytest <pytest_>`_ is used to run tests. 
+
+The name of a ``pytest`` file must be in the form ``test_XXX.py``, where ``XXX``
+is either the function, method, or class that you are testing or some other explicative
+name. In the command line, the ``-k`` argument can be used to filter the tests to run.
+For more information, see `pytest usage <pytest_usage_>`_.
+
+Here are some guidelines for creating good unit tests: 
+
+- Assign long and descriptive names to tests.
+- Use the `Codecov <codecov_>`_ tool to ensure all implemented code is tested.
+- Check that tests return the same results each time. 
+- Verify that tests are independent.
+- Write tests that verify only one part of the code at a time.
+- Make tests as short and fast as possible.
+
+`What makes a good unit test <article_good_unit_test_>`_ 
+is an exhaustive list of tips for creating good unit tests.
+
+Most PyMAPDL tests require a connection to a running instance of
+MAPDL, which makes them integration tests. If your test
+requires a running MAPDL instance, you can use the PyMAPDL
+`mapdl <mapdl_fixture_>`_ method in your function signature.
+It is executed upstream of each test and not within all tests.
+
+.. code:: python
+
+   def test_my_new_feature(mapdl):  # pass the 'mapdl' fixture as an argument.
+       mapdl.prep7()
+       # .... more code
+
+       return True  # if everything goes ok until here
+
+
+If you do not have MAPDL installed locally but still want to run the
+unit testing, you must set up the following environment variables.
+
+In Windows, use:
+
+.. code:: pwsh-session
+
+    SET PYMAPDL_START_INSTANCE=False
+    SET PYMAPDL_PORT=<MAPDL Port> (default 50052)
+    SET PYMAPDL_IP=<MAPDL IP> (default 127.0.0.1)
+
+In Linux, use:
+
+.. code:: console
+
+    export PYMAPDL_START_INSTANCE=False
+    export PYMAPDL_PORT=<MAPDL Port> (default 50052)
+    export PYMAPDL_IP=<MAPDL IP> (default 127.0.0.1)
+
+This tells ``ansys.mapdl.core`` to attempt to connect to the existing
+MAPDL service by default when the ``launch_mapdl`` function is used.
+
+Additionally you can use the environment variables ``PYMAPDL_MAPDL_EXEC`` 
+and ``PYMAPDL_MAPDL_VERSION`` to specify MAPDL executable path and the
+version to launch (if multiple versions of MAPDL are installed).
```

Comment:
```suggestion
Additionally, you can use the ``PYMAPDL_MAPDL_EXEC`` and ``PYMAPDL_MAPDL_VERSION``
environment variables to specify the MAPDL executable path and the version to launch (if
multiple versions of MAPDL are installed).
```

Reply:
Pinging @PipKat , because we should use 
> :envvar:`myvar`

Reply:
@germa89 Was this meant to be informational? I'm not sure if you want me to follow up on something here!

Reply:
Just to let you know that when mentioning environment variables we should use `:envvar:'myenvvar'`

Just that :)